src/bit-systems/object-menu.ts

```diff
@@ -10,12 +18,19 @@ import type { EntityID } from "../utils/networking-types";
 import { setMatrixWorld } from "../utils/three-utils";
 import { deleteTheDeletableAncestor } from "./delete-entity-system";
 import { createMessageDatas, isPinned } from "./networking";
+import { paths } from "../systems/userinput/paths";
+import { TRANSFORM_MODE } from "../components/transform-object-button";
+import { ScalingHandler } from "../components/scale-button";
 
 // Working variables.
-const _vec1 = new Vector3();
-const _vec2 = new Vector3();
+const _vec3_1 = new Vector3();
+const _vec3_2 = new Vector3();
+const _quat = new Quaternion();
 const _mat4 = new Matrix4();
 
+// TODO: Avoid any
```

Comment:
Why can't this be of type `ScalingHandler`?
```javascript
let scalingHandler: ScalingHandler | null = null;
```

Reply:
Good catch, updated.